provider/trips.json

```diff
@@ -322,25 +322,20 @@
               },
               "start_time": {
                 "$id": "#/properties/data/properties/trips/items/properties/start_time",
-                "type": "number",
                 "description": "The time the trip began, expressed as a Unix Timestamp",
-                "default": 0.0,
-                "examples": [
-                  1529968782.421409
-                ]
+                "$ref": "#/definitions/timestamp"
               },
               "end_time": {
                 "$id": "#/properties/data/properties/trips/items/properties/end_time",
-                "type": "number",
                 "description": "The time the trip ended, expressed as a Unix Timestamp",
-                "default": 0.0,
-                "examples": [
-                  1531007628.3774529
-                ]
+                "$ref": "#/definitions/timestamp"
               },
               "parking_verification_url": {
                 "$id": "#/properties/data/properties/trips/items/properties/parking_verification_url",
-                "type": "string",
+                "type": [
+                  "string",
+                  "null"
+                ],
```

Comment:
These changes should all be in `0.2.x` too, just clarifying what `null` means and what's allowed.